tests/Functional/Controller/Admin/SecurityControllerTest.php

```diff
@@ -91,7 +102,16 @@ public function testDeleteAction(): void
 
     public function testRolesAction(): void
     {
-        /* @todo This test is incomplete. */
-        $this->markTestIncomplete();
+        // Arrange
+        $localUser = $this->em->getRepository(LocalAccount::class)->findAll()[0];
+        $id = $localUser->getId();
+
+        // Act
+        $crawler = $this->client->request('GET', "/admin/security/{$id}/roles");
+        $this->assertEquals(200, $this->client->getResponse()->getStatusCode());
+        $form = $crawler->selectButton('Opslaan')->form();
+        $form['form[admin]']->setValue(false);
+        $this->client->submit($form);
+        $this->assertSelectorTextContains('.container', 'Rollen bewerkt');
```

Comment:
This should also check whether the roles actually changed in database

Reply:
true, changed it to check for that as well